### ISSUE TITLE
PyTorch and TensorFlow training workflows no longer show in UI.

### ISSUE THREAD

**Issue by Vafilor** [onepanelio/core#614]
Right now the labels column are null and the query selector for no labels stills an empty json object `{}`. This makes the result set empty.

As a fix, if there are no labels in the filter, don't apply any.
